src/status_im/group_chats/core.cljs

```diff
@@ -115,11 +115,9 @@
                                 :dsts          members
                                 :success-event [:transport/message-sent
                                                 chat-id
-                                                (transport.utils/message-id
-                                                 ;; NOTE: There is no clock-value here.
-                                                 ;; Will we have collision?
-                                                 {:from current-public-key
-                                                  :chat-id chat-id})
+                                                (transport.utils/message-id (assoc (:message payload)
```

Comment:
what is it using from the payload exactly? The `message-id` function is using destructuring and only takes `chat-id` `from` and `clock-value` from the map so I assume the only thing that it's using here from the message payload is `clock-value`?

Reply:
never checked actually,  but looks right, yes

Reply:
could you just pass the `clock-value` to the function then?

Reply:
I'd rather not to be honest, I think at this layer we should not care which fields we pass (we pass a well formed `message`), the interface of the function just takes a message and returns a `message-id` (say we change to include an extra field or whatever, this code should ideally not change)

Reply:
ok